saleor/core/utils/random_data.py

```diff
@@ -505,14 +508,14 @@ def create_fulfillments(order):
 
 
 def create_fake_order(discounts, max_order_lines=5):
-    user = random.choice(
-        [None, User.objects.filter(is_superuser=False).order_by("?").first()]
-    )
-    if user:
-        address = user.default_shipping_address
+    customers = User.objects.filter(is_superuser=False, is_staff=False).order_by("?")
```

Comment:
Maybe we should have a few orders for staff users(Not for superuser) in random data?

Reply:
Maybe yes, but not sure if this brings much value to the demo shop?

Reply:
This function is used in `populatedb` on local instances of Saleor. We should have orders for staff users for the purpose of testing Saleor. It's allowed to find edge cases in the development process.

Reply:
Good point, I'll change it then.